File: Golang/buble_sort.go
package main

import "fmt"

func main() {
	data := []int{21, 2, 5, 1, 3, 4}
	fmt.Println("Data int not sort", data)
	BubbleSort(data)
	fmt.Println("Data int sort", data)
}

func BubbleSort(data []int) {
	for i := 0; i < len(data); i++ {
		for j := 1; j < len(data)-i; j++ {
			if data[j] < data[j-1] {
				data[j], data[j-1] = data[j-1], data[j]
			}
		}
	}
}

File: Python/buble_sort.py
#Bubble Sort Algorithm
    def bubbleSort(data):        lenght = len(data)
        for iIndex in range(lenght):            swapped = False
            for jIndex in range(0, lenght - iIndex - 1):
                if data[jIndex] > data[jIndex + 1]:                    data[jIndex], data[jIndex + 1] = data[jIndex + 1], data[jIndex]                    swapped = True
            if swapped == False:                break
        print(data)
